hoot-core/src/main/cpp/hoot/core/io/OgrWriter.cpp

```diff
@@ -110,7 +110,7 @@ OgrWriter::OgrWriter():
 
   _textStatus = ConfigOptions().getWriterTextStatus();
   _includeDebug = ConfigOptions().getWriterIncludeDebugTags();
-  _maxFieldWidth = 10240; // Default to a max of 10kb per attribute. This gets reduced if needed
+  _maxFieldWidth = -1; // We set this if we really need to.
```

Comment:
Do we set this elsewhere?  Trying to understand how max width will be used with this value (-1).

Reply:
We only set this if we have to set the max width for a text field. Currently, the only format that sets this is Shapefile,
```
    if (name == QString("ESRI Shapefile"))
    {
      options["ENCODING"] = "UTF-8";
      _maxFieldWidth = 254; // Shapefile DBF limit
    }
```

From the GDAL docs, https://www.gdal.org/drv_shapefile.html
* String fields without an assigned width are treated as 80 characters.
* Starting with GDAL/OGR 1.10, the driver knows to auto-extend string and integer fields (up to the 255 bytes limit imposed by the DBF format) to dynamically accommodate for the length of the data to be inserted.

I am not sure why the behavior from the second bullet doesn't seem to be working.